packages/coinstac-simulator/src/index.js

```diff
@@ -99,4 +100,7 @@ module.exports = {
       });
     });
   },
+
+  // load in file loading utlilities
+  fileLoader,
```

Comment:
What do you think of exposing all of the file loading methods on the root export? This seems easier:

```js
const cstacSim = require('coinstac-simulator');

module.exports = {
  local: [{
    x: cstacSim.loadFiles('./path/to/sweet/*.csv'),
    y: cstacSim.loadFile('./path/to/other/*.stuff'),
  }],
};
```